geode-management/src/main/java/org/apache/geode/management/api/RuntimeResponse.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.geode.management.api;
+
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+
+/**
+ * the response object for all CMS requests must adhere to this interface to support injection of
+ * common attributes
+ */
+@JsonTypeInfo(use = JsonTypeInfo.Id.CLASS, property = "class")
+public interface RuntimeResponse {
+  String getGroup();
```

Comment:
I don't see the connection between "RuntimeResponse" with "group". "group" is a concept in "CacheElement", Why these two methods need to be declared here?

Reply:
The concept of CacheElement has been eliminated.  I wanted the response class to be completely unconstrained, but unfortunately we have logic to inject group information in the CMS layer, so this interface captures the minimum obligations of the response type to facilitate this injection.  For response types that are built on CacheElement these are satisfied "for free" but when we start adding request and response types that aren't associated with a CacheElement, they would need to implement this, or perhaps better we would need to look at eliminating this injection requirement.

Reply:
Is there any reason why request/response not being a CacheElement? If you put "group" concept in a RuntimeResponse, then it essentially becomes the "new" CacheElement. The CacheElement started with a simply a marker interface that's only serializable and has an Id and group.

Reply:
changed everything back to CacheElement for this PR.  I will create a separate PR to explore additional changes like RuntimeResponse.